README.md

```diff
@@ -1,7 +1,9 @@
-# Ontology Access Kit
+# Ontology Access Kit (OAK)
 
 Python lib for common ontology operations over a variety of backends.
 
+<img src="https://github.com/jmcmurry/closed-illustrations/blob/master/logos/oak-logos/oak-logo_black-banner.png" width="20%">
```

Comment:
should we put this in the docs dir here in case e.g Julie changes here directory layout?

Reply:
yeah, i wasn't sure whether it was better to link to it or download it and add it to this repo (which also seems wrong, in a way -- what if she updates it or something)?